(Hopefully) fixes various problem in the documentation (#10)
vimseo/vimseo: docs/tutorials/vvuq_study/solution_verification/solution_verification_method.md

<!--
 Copyright 2021 IRT Saint Exupéry, https://www.irt-saintexupery.com

 This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
 International License. To view a copy of this license, visit
 http://creativecommons.org/licenses/by-sa/4.0/ or send a letter to Creative
 Commons, PO Box 1866, Mountain View, CA 94042, USA.
-->

<!--
 Copyright (c) 2015 IRT-AESE.
 All rights reserved.

 Contributors:
    INITIAL AUTHORS - API and implementation and/or documentation

>         :author: XXXXXXXXXXX

Author: ```
        :author: Sebastien BOCQUET
```

    OTHER AUTHORS   - MACROSCOPIC CHANGES
-->

## Solution verification

Solution verification can be carried out by analyzing model results on progressively refined meshes.
The two main indicators to assess mesh convergence are the
Relative Discretization Error (RDE) [@roy2005_review_verif]

> and the Richardson extrapolation. The latter is mostly used in Computation Fluid Dynamics, and

Author: and the Richardson extrapolation. The latter is primarily used in Computation Fluid Dynamics, and

> assumes that the model is in the asymptotic range regarding the mesh size.

Author: assumes that the model is in the asymptotic range with respect to the mesh size.

In [@krysl2022_confidence_intervals_richardson], it is shown that using four meshes allows

> to estimate the confidence in Richardson extapolation.

Author: to estimate the confidence in Richardson extrapolation.

!!! note

    The time (for explicit models) or pseudo-time integration scheme could also be verified.

> These two methods are implemented in VIMS, in the tool named ``DiscretizationSolutionVerification``.

Author: ```
These two methods are implemented in VIMSEO, in the tool named ``DiscretizationSolutionVerification``.
```

> and can be displayed by the dashboard ``dashboard_verification``.

Author: ```suggestion
and can be displayed by the method ``DiscretizationSolutionVerification.dashboard_verification``.
```
 ?

Reviewer: `dashboard_verification` is a command line interface. So I think it can be left as is.

Solution verification is performed through a convergence study of the solution
on four progressively refined meshes (only three meshes would be strictly necessary),

> which allows to compute some uncertainties on the $GCI$, RDE and

Author: ```
on four progressively refined meshes (only three meshes would be strictly necessary).
This allows for the computation of  some uncertainties on the $GCI$, RDE and
```

Richardson extrapolation.

> An example of Python script setting up the solution verification for the ``OpfmPlate`` model with load case ``PST``

Author: ```
An example of Python script for setting up the solution verification for the ``OpfmPlate`` model with load case ``PST``
```

$q_{\gamma_i}$ is the solution obtained on the mesh with element size $\gamma_i$.
$E_q = q_{exact} - q_{\gamma}$ is the true error of the solution.

#### Richardson extrapolation

Considering $\gamma$ is a continuous variable, the true error on the simulation result
$q$ can be expanded in $\gamma$:

$$
E_q(\gamma) = q_{exact} - q_{\gamma} \approx C {\gamma}^{\beta}
$$

$\beta$ can be obtained by solving a non-linear equation involving
solutions on three progressively refined meshes.
Note that a solution for $\beta$ cannot always be found.

Once $\beta$ is found, $C$ can be obtained by considering the solution on two meshes
(by subtracting the above equation, which cancels $q_{exact})$.
Finally, the Richardson extrapoled value $q_{exact}$ can be computed from $C$ and $\beta$.

#### Grid Convergence Index

The $GCI$ is an indicator of the solution convergence on the mesh.
It requires to know the order of convergence of the solution,
based on the computation of the extrapoled value with Richardson method.
The $GCI$ requires the solution on two meshes. Thus different values
for the $GCI$ can be obtained from four meshes.

> For the two fines meshes, the $GCI$ is:

Author: ```
For the two finest meshes, the $GCI$ is:
```

$$
GCI_{finest} = F_s \frac{q_{\gamma_3} - q_{\gamma_2}}{q_{\gamma_3}} \frac{1}{(\gamma_2 /
\gamma_3)^{\beta} - 1}
$$

> An error band can be computed for the $GCI$[@krysl2022_confidence_intervals_richardson]:

Author: ```suggestion
A confidence interval can be computed for the $GCI$[@krysl2022_confidence_intervals_richardson]:
```
?

Reviewer: Ok, thanks

$E_b(GCI_{finest}) = q_{\gamma_3} (1 \pm GCI_{finest})$

> However, it is often the case that Richardson extrapolation cannot be computed

Author: However, Richardson extrapolation is often not computable

(for instance, if the solution oscillates between consecutive refined meshes).

> In such case, we can still provide a value of the $GCI$ by assuming a convergence order.

Author: ```
In such cases, we can still provide a value for the $GCI$ by assuming a convergence order.
```

Note also that two meshes must be selected to compute this indicator.
Currently, the $GCI$ is computed on the two finest meshes and the two coarsest meshes.
The solution verification dashboard shows these two $GCI$ values for:

- the computed order of convergence, if possible
- an assumed order of convergence of one and two.

#### Relative Discretization Error

The RDE is another indicator of the numerical error due to
the mesh discretization.

> It can be computed from two meshes. Thus three RDE values can be computed from the four simulated meshes.

Author: It can be computed from two meshes. Therefore, three RDE values can be computed from the four simulated meshes.

> Note that it also requires that the extrapolated value with Richardson method is computed.

Author: Note that it also requires the extrapolated value using the Richardson method to be computed.
